diff --git a/assets/js/login.js b/assets/js/login.js
index dae7fa1..510681b 100644
--- a/assets/js/login.js
+++ b/assets/js/login.js
@@ -20,7 +20,7 @@ import 'bootstrap';
 
             if ($emailInput.val().length >= 30) {
                 import('./Components/show_long_email_error').then(showLoginEmailError => {
-                    showLoginEmailError($emailInput);
+                    showLoginEmailError.default($emailInput);
                 });
             }
         });
