diff --git a/assets/js/Components/RepLogApp.js b/assets/js/Components/RepLogApp.js
index 612505e..2dc09cb 100644
--- a/assets/js/Components/RepLogApp.js
+++ b/assets/js/Components/RepLogApp.js
@@ -1,6 +1,6 @@
 'use strict';
 
-const Helper = require('./RepLogAppHelper');
+import Helper from './RepLogAppHelper';
 const $ = require('jquery');
 
     let HelperInstances = new WeakMap();
@@ -209,4 +209,4 @@ const $ = require('jquery');
 </tr>
 `;
 
-module.exports = RepLogApp;
+export default RepLogApp;
diff --git a/assets/js/Components/RepLogAppHelper.js b/assets/js/Components/RepLogAppHelper.js
index 29d7568..8682e3e 100644
--- a/assets/js/Components/RepLogAppHelper.js
+++ b/assets/js/Components/RepLogAppHelper.js
@@ -1,4 +1,4 @@
-module.exports = class Helper {
+export default class Helper {
     constructor(repLogs) {
         this.repLogs = repLogs;
     }
diff --git a/assets/js/rep_log.js b/assets/js/rep_log.js
index 688ba06..29b33e2 100644
--- a/assets/js/rep_log.js
+++ b/assets/js/rep_log.js
@@ -1,4 +1,4 @@
-const RepLogApp = require('./Components/RepLogApp');
+import RepLogApp from './Components/RepLogApp';
 const $ = require('jquery');
 
 $(document).ready(function() {
